scrapers/covid-19/historical.js

```diff
@@ -77,6 +77,29 @@ const historicalV2 = async (keys, redis) => {
 			: parsedAtIndex['Province/State'].toLowerCase();
 		return newElement;
 	});
+	/* This is an edge case to handle recovered data for Canada.
+	   Since JHU dataset doesn't have recovered data for Canada
+	   disaggregated by Province/Region, the aggregate data is
+	   lost when recovered data is formatted in formatRecoveredData
+	   function. This  will create a non-existent Province/Region
+	   called recovered-aggregate with cases and deaths set to 0
+	   and recovered is set to aggregate recovered data. This data
+	   is pulled only if historical data for Canada is requested.
+	*/
+	for (let i = 0; i < result.length; i++) {
+		if (result[i].country.toLocaleLowerCase() === 'canada') {
+			const data = { country: result[i].country, countryInfo: { ...result[i].countryInfo }, province: 'recovered-aggregate', timeline: { cases: {}, deaths: {}, recovered: {} } };
```

Comment:
If you do the above comment this will already be here for you to make a deep copy of the `canadaData` object
```js
data = JSON.parse(JSON.stringify(canadaData);
```